src/NServiceBus.AcceptanceTests/Sagas/When_finder_exists_and_found_saga.cs

```diff
@@ -11,14 +11,18 @@
     using Persistence;
 
     [TestFixture]
-    public class When_a_finder_exists_and_found_saga : NServiceBusAcceptanceTest
+    public class When_finder_exists_and_found_saga : NServiceBusAcceptanceTest
```

Comment:
I wanted to keep all finder related tests a bit closer by starting with the same naming (When_finder), so how about When_finder_returns_existing_instance?